contracts/actions/common/uSwap.sol

```diff
@@ -211,6 +211,8 @@ contract uSwap {
 
   function swapTokens(SwapData calldata swapData) public returns (uint256) {
     IERC20(swapData.fromAsset).safeTransferFrom(msg.sender, address(this), swapData.amount);
+    uint256 balanceOnContract = IERC20(swapData.fromAsset).balanceOf(address(this));
+
```

Comment:
Is it used?

Reply:
Nah, sorry was still working on this branch